eval/eval.py

```diff
@@ -595,8 +590,7 @@
 
 parser = argparse.ArgumentParser(
     description=(
-        "This program evaluates the spelling and grammar checking performance "
-        "of GreynirCorrect on iceErrorCorpus"
+        "This program evaluates the spelling and grammar checking performance " "of GreynirCorrect on iceErrorCorpus"
```

Comment:
The formatter really fucked this one up.

Reply:
Fixed it